packages/babel-core/src/index.ts

```diff
@@ -1,3 +1,13 @@
+if (!process.env.IS_PUBLISH) {
```

Comment:
I personally prefer to put it in `jest.config.js`.

Reply:
I put it here because in local development I often import `@babel/core` directly, to see&debug behavior. This is stripped anyway on publish, but having the check here makes it work both with and without Jest in local development.